applications/IgaApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -20,14 +20,204 @@
 #include "processes/process.h"
 #include "custom_python/add_custom_utilities_to_python.h"
 
-#include "spaces/ublas_space.h"
-#include "linear_solvers/linear_solver.h"
+#include "iga_application_variables.h"
+#include "custom_utilities/node_curve_geometry_3d.h"
+#include "custom_utilities/node_surface_geometry_3d.h"
 
 namespace Kratos {
 namespace Python {
 
-void  AddCustomUtilitiesToPython(pybind11::module& m)
+void AddCustomUtilitiesToPython(pybind11::module& m)
 {
+    namespace py = pybind11;
+    using namespace pybind11::literals;
+
+    using Array3D = Kratos::array_1d<double, 3>;
+
+    // register Interval
```

Comment:
This bracket defines what?

Reply:
I said because is commented

Reply:
I just use the brackets to create a scope for each class registraction. The comment at the beginning is just a description of the block.

```
// register SurfaceGeometry
{
    using Type = ANurbs::SurfaceGeometry3D; // I can declare such things in each block
    using Pointer = std::shared_ptr<Type>;

    ...
}
```

Reply:
Ok